gc_tests/valgrind.supp

```diff
@@ -2,12 +2,6 @@
    <uninitialized_stack>
    Memcheck:Cond
    ...
-   fun:scan_stack
+   fun:_ZN8gcmalloc2gc9Collector13check_pointer*
```

Comment:
I don't know the mangling rules, but I was sort of expecting something like `*Collector*_pointer` or similar? Obviously I can see problems with the wildcards, so maybe it's a bad idea!